config/spl-build.m4

```diff
@@ -1476,3 +1477,26 @@ AC_DEFUN([SPL_AC_MUTEX_OWNER], [
 	])
 	EXTRA_KCFLAGS="$tmp_flags"
 ])
+
+dnl #
+dnl # 4.7 API change
+dnl # i_mutex is changed to i_rwsem. Instead of directly using
+dnl # i_mutex/i_rwsem, we should use inode_lock() and inode_lock_shared()
```

Comment:
Technically `inode_lock()` was introduced in 4.5 and `inode_lock_shared()` in 4.7.  But I think it does make sense to simply this and group them since if they have `inode_lock_shared()` they'll also have `inode_lock()`.